server/core/config/local.py

```
@@ -1,9 +1,51 @@
-    'http://localhost:4321',
-]
\ No newline at end of file
+]
+
+CORS_ALLOW_CREDENTIALS = True
+
+# Database settings (you might want to use SQLite for local development)
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': config('DB_NAME', default='your_local_db_name'),
```

Ruff (0.8.2)

`config` may be undefined, or defined from star imports

(F405)